pkgs/applications/graphics/ImageMagick/7.0.nix

```diff
@@ -62,7 +63,7 @@ stdenv.mkDerivation {
 
   propagatedBuildInputs =
     [ bzip2 freetype libjpeg lcms2 ]
-    ++ lib.optionals (!stdenv.hostPlatform.isMinGW)
+    ++ lib.optionals (x11Support && !stdenv.hostPlatform.isMinGW)
       [ libX11 libXext libXt libwebp ]
```

Comment:
`libwebp` is only needed for X11 support?

Reply:
you made me thinking - looking at it, seems like it shouldn't even be here in the first place.

Reply:
It's definitely used: https://github.com/ImageMagick/ImageMagick/blob/7.0.11-3/configure.ac#L2851

Reply:
my point was rather about `libwebp` being part of this optional

Reply:
I guess it's not available on MinGW for some reason.